tests/index.js

```diff
@@ -732,6 +732,36 @@ describe('Thorn', () => {
 
                 expect(server.isDone()).to.be.true;
             });
+
+            it('should use the internal username for server interactions', function*() {
```

Comment:
Nitpick: can we drop the `the` portion?
Something like`it('should use internal usernames for server interactions', ...`?